test/FunctionalTests/Microsoft.Extensions.OData.Migration.Tests/InlineCountTranslationTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿// ------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright © Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Extensions.OData.Migration.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Xunit;
+
+    public class InlineCountTranslationTest
+    {
+        private static readonly Uri serviceRoot = new Uri("http://localhost:80/");
+        private static readonly ODataMigrationMiddleware middleware = TestModelProvider.ODataSvcSampleMiddleware(serviceRoot);
+
+        [Theory]
+        [MemberData(nameof(InlineCountTranslationQueries))]
+        public void TestInlineCountTranslations(string name, string testQuery, string expectedQuery)
```

Comment:
>name [](start = 55, length = 4)

where's using "name"? same for other test cases

If that's only for the comments, just add them as comments into the test case. for example: 

{  new object[] { "Products?$inlinecount=allpages", "Products?$count=true" } }, // All pages should become true

Reply:
change name to comments